docs/guides/development/nodejs/nodejs-twitter-bot/index.md

```diff
@@ -0,0 +1,294 @@
+	        } else {
+	            console.log('Tweeted: ' + reply.text);
+	        }
+	    };
+	}
+```
+</br>
+
+`pressStart` contains a few local variables, a bit of logic, and a final function that must be included in the `T.post` method. You can use an unnamed function there and it will do the same thing, but I went ahead and wrote on separately for readability. Essentially, the function `gameOver` gives us a chance to log an error if it occurs or to log the tweet that was sent out.
+
+`pressStart` takes “tweet” as a parameter. This is the tweet that another user writes that tags SnesSoundtracks. That tweet has tons of data attached to it, data that Twit helps us parse through. The first three variables are the id of the tweet, the text of the tweet, and the username of the person who wrote the tweet. We will need those three in order to respond accurately as a comment to the original tweet by the other user.
+
+Up next is a regex for whatever word you want to activate the bot to reply. I chose “please." As long as the user is polite, they’ll get a random soundtrack.
+
+The regex has “g” and “i” at the end so it ignores capitalization and checks globally for the word please. `playerOne` is a variable that can either be an empty array or will use `.match` to create an array with one element: the word “please”. `playerTwo` is a boolean that verifies whether the array `playerOne` has anything in it.
+
+The if statement requires that the tweet text contains the bot’s name and an array of at least one element was passed into `playerTwo`. If both of these come back as true, then we proceed to an area where the variable `replyText` is created, which includes a random element of the array, as well as the username of the person being replied to and a short message. `replyText` is passed into an object that contains two properties: `status` and `in_reply_to_status_id`. `status` is the actual text to be posted to the tweet, in our case the variable `replyText` is our status. `In_reply_to_status_id` is defined as `id`, which is a variable from the beginning of the `pressStart` function. `id` is a unique identifier of a tweet from Twitter’s API. This allows Twit to identify which tweet the bot will reply to as a comment. Finally, the else statement at the end will log a quote from Jurassic Park to the console if the user doesn’t say please, just so we can see why the bot didn't tweet back.
+
+```javascript
+	function pressSelect() {
+
+	    var weeklyReplyText = soundtrackArray[soundtrackArrayElement] + " Here's your soundtrack for the week!";
+	    T.post('statuses/update', { status: weeklyReplyText }, gameOver2);
+
+	    function gameOver2(err, reply) {
+	        if (err) {
+	            console.log(err.message);
+	            console.log("Game Over");
+	        } else {
+	            console.log('Tweeted: ' + reply.text);
+	        }
+	    }
+	}
+
+	 const job1 = schedule.scheduleJob(rule, pressSelect);
+
+	 job1.on("Every Day Tweet", pressSelect);
```

Comment:
In my testing, it didn't seem like this extra event listener actually did anything. The previous  `const job1 = schedule.scheduleJob(rule, pressSelect);` line seemed to do everything needed to schedule the tweet, which matched up with what i read from the node-schedule documentation. So, I removed the ` job1.on("Every Day Tweet", pressSelect);` line